bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
 #----alsa install
 sudo apt-get install -y alsa-base alsa-utils
+
+#----install pigpio (npm install pigpio will be exec by tjbotlib package.json)
+curl abyz.co.uk/rpi/pigpio/pigpio.zip > pigpio.zip
```

Comment:
hmmm.... I'm following the official installation steps:
https://www.npmjs.com/package/pigpio

I can test with `sudo apt-get install -y` to see if it will works.